meta/TreeMasses.m

```diff
@@ -524,6 +535,11 @@
 IsZBoson[sym_[___]] := IsZBoson[sym];
 IsZBoson[field_Symbol] := field === GetZBoson[];
 
+IsGluon[Susyno`LieGroups`conj[sym_]] := IsGluon[sym];
```

Comment:
Is this function exported? I think it may be a good idea to do so.

Reply:
Hi @Expander, what do you mean by exported?  Thanks.

Reply:
Sorry, I meant a `IsGluon::usage = "..."` statement at the top.

Reply:
Okay, I have included usage text for all the functions that I need for this addition.